src/ParallelRun.jl

```diff
@@ -538,35 +499,6 @@ function get_overlapping_fields(query::BoundingBox, stagedir::String)
 end
 
 
-"""
-called from main entry point for inference for one field
-(used for accuracy assessment, infer-box is the primary inference
-entry point)
-"""
-function infer_field(rcf::RunCamcolField,
-                     stagedir::String,
-                     outdir::String;
-                     joint_infer=false,
-                     objid="")
```

Comment:
This function does almost nothing, and it's only used by `validate_on_stripe82.jl` so I just did what this function used to do directly in `validate_on_stripe82.jl`.